hawc/apps/mgmt/signals.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+from django.apps import apps
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from . import models
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(post_save, sender=models.TaskType)
+def default_configuration(sender, instance, created, **kwargs):
+    """
+    Update study management tasks when a new task type is added
```

Comment:
This signal is not linked to task progress - just triggers new task creation for now when a task type is created.